### ISSUE TITLE
Add option to use real Lego colors 

### ISSUE THREAD

**Issue by kpalmvik** [Wildhoney/Legofy#8]
It should be possible to specify that transform() should only render bricks in the official Lego brick colors (http://lego.wikia.com/wiki/Colour_Palette)

This is probably tricky to do using the current implementation, and may require some additional image processing.

See https://github.com/JuanPotato/Legofy/issues/2 for reference.

**Comment by Wildhoney** [comment]
Agreed that it should definitely be possible &mdash; others have also mentioned applying dithering and then setting up a LEGO shopping basket with the bricks that make up your image.

Will look into this one :+1:

**Comment by Wildhoney** [comment]
In fact looking at the quantize library [RgbQuant](https://github.com/leeoniya/RgbQuant.js/tree/master), this should be able to do it for us using the `palette` option.
